x-pack/plugins/observability_solution/apm/public/components/shared/ml_callout/index.tsx

```diff
@@ -106,7 +106,7 @@ export function MLCallout({
         primaryAction: isOnSettingsPage ? (
           <EuiButton
             data-test-subj="apmMLCalloutUpdateJobsButton"
-            color="success"
+            color="accentSecondary"
```

Comment:
Generally the design decision is that there should be no `accentSecondary` buttons.
Maybe the responsible designer might have a look, what it could be instead?

If that's not possible, we can keep it for now and would need to circle back to it later.